spotlight-client/auth0/hooks/02-add-app-metadata-to-idtoken.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Handler that will be called during the execution of a PostLogin flow.
+ *
+ * @param {Event} event - Details about the user and the context in which they are logging in.
+ * @param {PostLoginAPI} api - Interface whose methods can be used to change the behavior of the login.
+ */
+exports.onExecutePostLogin = async (event, api) => {
+  const namespace = "https://recidiviz.org"; // namespace has to be an HTTP URL
+  // https://auth0.com/docs/actions/triggers/post-login
+  // https://auth0.com/docs/get-started/apis/scopes/sample-use-cases-scopes-and-claims
+  // https://auth0.com/docs/secure/tokens/json-web-tokens/create-namespaced-custom-claims
+  api.idToken.setCustomClaim(
+    `${namespace}/app_metadata`,
+    event.user.app_metadata
```

Comment:
So I see here that you're adding the `app_metadata` to the token, but where does the `state_code` get added to the `app_metadata`? I don't see any actions to do that in Auth0.

Reply:
Someone needs to go into the recidiviz-spotlight-staging Auth0 tenant, look up the user, and add `state_code` to the user's `app_data`. I'm thinking of documenting these steps.

Reply:
Just added a paragraph explaining this in the README